os/board/rtl8721csm/src/component/common/api/wifi_interactive_mode.c

```diff
@@ -755,16 +767,41 @@ int8_t cmd_wifi_on(WiFi_InterFace_ID_t interface_id)
 
 int8_t cmd_wifi_off(void)
 {
+	int mode;
+
 #if CONFIG_WEBSERVER
 	stop_web_server();
 #endif
 #if CONFIG_ENABLE_P2P
 	cmd_wifi_p2p_stop(0, NULL);
 #else
+	if((rltk_wlan_running(WLAN0_IDX) == 0) &&

```

Comment:
0 means OK or FAIL? Please change this to return value if there is a defined return type like enum. If not, How about making `#define SUCCESS/FAIL  0` and using it?